test/Microsoft.DotNet.Cli.Sln.Internal.Tests/Microsoft.DotNet.Cli.Sln.Internal.Tests.cs

```diff
@@ -496,7 +496,15 @@ public void WhenGivenASolutionWithInvalidPropertySetItThrows()
             };
 
             action.ShouldThrow<InvalidSolutionFormatException>()
-                .WithMessage("Invalid format in line 7: Property set is missing '.'");
+                .WithMessage(FormatError(7, LocalizableStrings.InvalidPropertySetFormatString, "."));
+        }
+
+        private static string FormatError(int line, string format, params object[] args)
```

Comment:
I did it this way to make it clear that the line is not just the first parameter of format, it belongs to a separate format string.